### ISSUE TITLE
Files are not removed from SVN

### ISSUE THREAD

**Issue by ocean90** [toolstack/git-to-wp-plugin-dir-release-script#13]
See this commit https://plugins.trac.wordpress.org/changeset/1532332/glotpress (ignore the commit message 🙄) and https://plugins.trac.wordpress.org/browser/glotpress/trunk/gp-includes/formats?rev=1532332

The files with the underscore should have been removed.

**Comment by ocean90** [comment]
That's because in https://github.com/toolstack/git-to-wp-plugin-dir-release-script/blob/13aae838faf6f6b567f3ba3dc46d6fc3a4556ee3/class.release.php#L505 the result of the `array_merge()` isn't stored in `$files`.

Only storing the file names is probably a bad idea since a file name can exist multiple times. `get_file_list()` should also ignore VCS directories like `.svn` or `.git`. With the above fix I currently have 5459 files for GlotPress because the VCS directories. 

But for some reasons the files are still not removed:

```
svn: E200005: '/private/var/folders/vr/z6wqt2xs6ys87x5rl53_xc900000gn/T/GWPd8jAPH/img' is not under version control
, imgsvn: E200005: Use --force to override this restriction (local modifications may be lost)
svn: E200005: '/private/var/folders/vr/z6wqt2xs6ys87x5rl53_xc900000gn/T/GWPd8jAPH/asc.gif' is not under version control
, asc.gifsvn: E200005: Use --force to override this restriction (local modifications may be lost)
svn: E200005: '/private/var/folders/vr/z6wqt2xs6ys87x5rl53_xc900000gn/T/GWPd8jAPH/desc.gif' is not under version control
, desc.gifsvn: E200005: Use --force to override this restriction (local modifications may be lost)
svn: E200005: '/private/var/folders/vr/z6wqt2xs6ys87x5rl53_xc900000gn/T/GWPd8jAPH/format_android.php' is not under version control
, format_android.phpsvn: E200005: Use --force to override this restriction (local modifications may be lost)
svn: E200005: '/private/var/folders/vr/z6wqt2xs6ys87x5rl53_xc900000gn/T/GWPd8jAPH/format_pomo.php' is not under version control
, format_pomo.phpsvn: E200005: Use --force to override this restriction (local modifications may be lost)
svn: E200005: '/private/var/folders/vr/z6wqt2xs6ys87x5rl53_xc900000gn/T/GWPd8jAPH/format_properties.php' is not under version control
, format_properties.phpsvn: E200005: Use --force to override this restriction (local modifications may be lost)
svn: E200005: '/private/var/folders/vr/z6wqt2xs6ys87x5rl53_xc900000gn/T/GWPd8jAPH/format_resx.php' is not under version control
, format_resx.phpsvn: E200005: Use --force to override this restriction (local modifications may be lost)
svn: E200005: '/private/var/folders/vr/z6wqt2xs6ys87x5rl53_xc900000gn/T/GWPd8jAPH/format_strings.php' is not under version control
, format_strings.php.
```

Fore more see https://gist.github.com/ocean90/db8a0b21b0f49cfb05e301924b4d47cb

**Comment by toolstack** [comment]
I've updated the function, the intent was to get a file list, relative to the working directories and then compare the differences.

Since it's a one sided compare (what is in SVN compared against what's in GIT) we don't have to worry about `.git` or other GIT specific files.

Likewise, we've already deleted the files/directories we don't want from the SVN through the `release.ini` file so `.svn` doesn't exist by the time we get to the deleted file list.
